artifacthub/library/general/disallowanonymous/1.0.1/samples/no-anonymous-bindings/constraint.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sDisallowAnonymous
+metadata:
+  name: no-anonymous
+spec:
+  match:
+    kinds:
+      - apiGroups: ["rbac.authorization.k8s.io"]
+        kinds: ["ClusterRoleBinding"]
+      - apiGroups: ["rbac.authorization.k8s.io"]
+        kinds: ["RoleBinding"]
+  parameters:
```

Comment:
Should we also have a version of the constraint featured in the suite that specifies no `allowedRoles`, as that was the issue we are trying to fix?

Reply:
Hi @julianKatz - I added 4 new test with no `allowedRoles ` to the rego unit tests: https://github.com/open-policy-agent/gatekeeper-library/pull/578/files#diff-f0536bb898d20b404acf2fce2427efc3874d5ad60e3e1873583247139cc8de2e

I suppose we could also add another to the gator test suite, but I'm not sure we can exclude that from inclusion in the documentation.